src/status_im/ui/components/list_item/views.cljs

```diff
@@ -1,81 +1,378 @@
+     [title-row
+      title-row-elements type icon? disabled?
+      theme subtitle content accessories])
+
+   (when (and subtitle (= :default type))
+     [subtitle-row subtitle-row-elements icon? theme])
+
+   (when content
+     [react/view {:margin-left (if icon? 2 0)}
+      (if (vector? content)
+        content
+        [content])])])
+
+(defn- accessories-column [accessories theme]
+  (let [last-accessory              (peek accessories)
+        last-accessory-is-component (and (not (stringify last-accessory))
+                                         (not= :chevron last-accessory))
+        second-last-accessory       (peek (pop accessories))]
+    (into
+     [react/view styles/accessories-container]
+     (for [accessory accessories]
+       (when-not (nil? accessory)
+         (with-meta
+           (cond
+             (= :chevron accessory)
+             [icons/icon :main-icons/next
+              {:container-style {:width           10
+                                 :height          16
+                                 :align-items     :center
+                                 :justify-content :center}
+               :resize-mode     :center
+               :color           (if (= theme :blue)
+                                  (colors/alpha colors/white 0.4)
+                                  colors/gray-transparent-40)}]
+
+             (= :check accessory)
+             [icons/icon :main-icons/check
+              {:color (if (= theme :blue)
+                        (colors/alpha colors/white 0.4)
+                        colors/gray)}]
+
+             (= :more accessory)
+             [icons/icon :main-icons/more
+              {:color (if (= theme :blue)
+                        (colors/alpha colors/white 0.4)
+                        colors/black)}]
+
+             :else
+             [react/view (cond-> {:margin-right (if (= accessory last-accessory) 0 16)}
+                           ;; `:chevron` container is 10px wide (see up)
+                           ;; but the chevron icon itself is 9px aligned in the
+                           ;; container to the right - so 1px white-space on left
+                           ;; that 1px + 15px margin makes 16px
+                           ;; as intended in design spec
+                           (= last-accessory :chevron)
+                           (assoc :margin-right 15))
+              (cond
+                (or (string? accessory) (keyword? accessory) (number? accessory))
+                [react/text {:style           (styles/accessory-text theme)
+                             :number-of-lines 1}
+                 (stringify accessory)]
+
+                (vector? accessory)
+                accessory
+
+                :else nil)])
+           {:key accessory}))))))
+
+;; every key is optional - use as needed
+;; combination of around 4 related keys are enough for most cases
+
+;; type
+;; :default (default), :small, or :section-header
+;; - :section-header
```

Comment:
is `:section-header ` same as https://github.com/status-im/status-react/blob/develop/src/status_im/ui/components/list_header/views.cljs#L5 ?

Reply:
it is exactly the same :) yes
I thought since we use `flat-list` quite a bit, it would be nice to feed the flat-list `data` as much as possible instead of occasional component in-between data, especially for flat-lists with `homogeneous list-items`. Then we can gracefully replace the use of `section-list` with `flat-list` which is more versatile.

Reply:
should I remove it and use `list_header` instead?